mungegithub/github/webhook.go

```diff
@@ -54,7 +56,13 @@ func (webhook WebHook) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 }
 
 // NewWebHookAndListen creates a new WebHook and listen to it
```

Comment:
Thats one reason. It also adds a level of indirection to the config option so that the raw key doesn't have to be put in the config file. I changed it to be handled the same way as the github api token.